### ISSUE TITLE
FR: V2 Truck Builder - Phase 2

### ISSUE THREAD

**Issue by shirin27** [hlxsites/vg-macktrucks-com#610]
From vg-macktrucks-com-rd created by [shirin27](https://github.com/shirin27): Netcentric/vg-macktrucks-com-rd#33  

**User story**
As a business owner, I want to be able to select different categories of the truck from the accordion, so that more content will appear explaining to me the different features of the truck.
As a business owner, I want to be able to click on different colour options to see the variety of truck colours available, so that I can choose my preference in the configurator. 

**Scope**
Accordion block for expanding information on the right side of the image
Selecting the colour on a tabbed column below the image, for the image to change

https://www.figma.com/file/oNwe2jHIICPMIe551EY62l/Mack-Reskin---Full-Project?node-id=2243%3A159073&mode=dev

**Acceptance Criteria** 

- All accordion categories are to the right 
  - Title of truck 
  - Clickable categories (Categories are in a lighter font colour) 
  - Truck Builder button below

- The location of colour selection is below the image to the left of the page
- Once a visitor clicks a colour - the image changes corresponding to the selected colour
- Mobile 
  - The accordion categories will appear below the image in a slide

**Desired view** 
![image](https://github.com/Netcentric/vg-macktrucks-com-rd/assets/158284953/9a2361ae-5628-442b-adc1-29ffbf776a94)
![image](https://github.com/Netcentric/vg-macktrucks-com-rd/assets/158284953/ce5279f1-8fc1-4528-b1e7-2dacd69767f2)

**Existing solution**
https://github.com/Netcentric/vg-macktrucks-com-rd/issues/114

- Accordion is to the right by default - can be changed to the left
- Photos can be inputted into the accordion

![image](https://github.com/Netcentric/vg-macktrucks-com-rd/assets/158284953/ef008e4b-6e2c-4a13-b0cc-ab64d4b1b279)

**Design Questions** 

- In the Figma there are only 4 colours, on the website there are 8 colours available
  - Which selection? Will editors have the option to add multiple colours (no min or max) ?
  - Will the colours to pick from be the same for one truck model (e.g. Granite) and its configurations (e.g. Standard Mixer/Tractor/Dump/…)? We need to know whether colours can be defined at the block level, or whether they need to be set per truck configuration
- Currently the name of the colour is displayed  
- What content to be displayed inside the accordion 
- Will there be drop down arrows on the accordion categories

**Comment by shirin27** [comment]
This will be part of MVP-2

**Comment by shirin27** [comment]
> Add the tabbed column where the image changes
> 
> https://www.figma.com/file/oNwe2jHIICPMIe551EY62l/Mack-Reskin---Full-Project?node-id=2243%3A159073&mode=dev

Can you please explain what exactly this means?

**Comment by shirin27** [comment]
@shirin27 Need clarity on why this is needed.

**Comment by shirin27** [comment]
@BeckyMedlin this ticket is a split from #32 , as the scope was too big when we initially refined the ticket.
I will clarify the exact scope of each ticket in a refinement session with the team, has the Truck configuration block become a priority? Should I bring it to refinement this week?
Tickets for Truck config block: #32 , #33 , #34

**Comment by shirin27** [comment]
@shirin27 No, it's still a priority 3. I asked @WendyKruger yesterday if the config blocks needed to be included with the Configurator work and she said no.

**Comment by WendyKruger** [comment]
@shirin27 is there still a design question here?
